src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -603,6 +612,20 @@ impl Node {
         frp.set_disabled.emit(false);
         Self {frp,model}
     }
+
+    fn error_color(error:&Option<Error>, style:&StyleWatch) -> color::Lcha {
+        use ensogl_theme::graph_editor::node::error as error_theme;
+
+        if let Some(error) = error {
+            let path = match *error.kind {
+                error::Kind::Panic    => error_theme::panic::color,
+                error::Kind::Dataflow => error_theme::dataflow::color,
+            };
+            style.get_color(path)
+        } else {
+            color::Lcha::transparent()
```

Comment:
Does it ean that we color error messages as transparent instead of hiding them using display object functions when errors are not provided? If so, we should hide errors by upsetting parents, so we are rendering less things then.

Reply:
The transparent color is for animations. But I added code for hinding when animation is complete.